.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
 - '0.12'
+- '4.1.1'
```

Comment:
Could you make this just `4.1` so we automatically build against latest bug fix line?